crates/protocol/src/operator/model.rs

```diff
@@ -7,7 +7,7 @@ use warg_crypto::hash::{DynHash, HashAlgorithm};
 use warg_crypto::signing;
 
 /// A operator record is a collection of entries published together by the same author
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
```

Comment:
I don't think changes to this file are necessary because we shouldn't be serializing records on the client, only the validation state.